tests/datastore/test_filesystem_datastore.nim

```diff
@@ -65,37 +67,37 @@ suite "FileSystemDatastore":
         ds.path(Key.init("a/b/c:d").get) == rootAbs / "a" / "b" / "c" / "d" & objExt
         ds.path(Key.init("a/b/c/d").get) == rootAbs / "a" / "b" / "c" / "d" & objExt
 
-  test "put":
+  asyncTest "put":
     let
       ds = FileSystemDatastore.new(root).get
       key = Key.init("a:b/c/d:e").get
       path = ds.path(key)
 
     var
       bytes: seq[byte]
-      putRes = ds.put(key, bytes)
+      putRes = await ds.put(key, bytes)
 
     check:
       putRes.isOk
       readFile(path).toBytes == bytes
 
     bytes = @[1.byte, 2.byte, 3.byte]
 
-    putRes = ds.put(key, bytes)
+    putRes = await ds.put(key, bytes)
 
     check:
       putRes.isOk
       readFile(path).toBytes == bytes
 
     bytes = @[4.byte, 5.byte, 6.byte]
 
-    putRes = ds.put(key, bytes)
+    putRes = await ds.put(key, bytes)
 
     check:
       putRes.isOk
       readFile(path).toBytes == bytes
 
-  test "delete":
+  asyncTest "delete":
```

Comment:
I wonder why you are using asyncTest==test. In Codex, all tests are using plain `test`

Reply:
See this comment: https://github.com/status-im/nim-datastore/pull/14#discussion_r911044894.

`test` coming from `pkg/asynctest` is: https://github.com/status-im/asynctest/blob/main/asynctest/templates.nim#L34-L37

Reply:
I'm not sure that he meant to define `asyncTest=test`. Asked him, let's wait for answer

Reply:
so it's the last thing to resolve. If you want, you can merge existing code and then fix it (if required) in next PR

Reply:
Let's merge it, and revisit `asyncTest=test` depending on his answer to your question.